lib/elixir/lib/macro.ex

```diff
@@ -159,10 +157,17 @@ defmodule Macro do
       expressions inside "blocks of code", which are either direct children
       of a `__block__` or the right side of `->`. The last expression of the
       block does not have metadata if it is not followed by an end of line
-      character (either a newline or `;`)
+      character (either a newline or `;`). This entry may appear multiple times
+      in the same metadata if the expression is surround by parens
+
+    * `:format` - set to `:keyword` when an atom is defined as a keyword
```

Comment:
```suggestion
    * `:format` - set to `:keyword` when an atom is defined as a keyword.
      It may also be set to `:atom` to distinguish `nil`, `false`, and `true`
```